downstairs/src/lib.rs

```diff
@@ -1062,8 +1058,13 @@ async fn proc_task(
     reply_channel_tx: mpsc::UnboundedSender<Message>,
 ) -> Result<()> {
     // Populate our state in the Downstairs, keyed by our `id`
-    let mut another_upstairs_active_rx =
-        ads.lock().await.new_connection(id, reply_channel_tx);
+    let (log, cancel_io) = {
+        let mut ds = ads.lock().await;
+        (
+            ds.log.new(o!("task" => "proc_task".to_string())),
```

Comment:
From a previous review comment, maybe here we add `ConnectionId` to the log line?

Reply:
Sure, done in 6dbde31b7be5